### PR TITLE
shayan/Drawing tools bug

### PR DESCRIPTION
**Pull request by shayan-binary-2**

### CODE REVIEW

File: src/Constant.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    channel:    { id: 'channel',     text: t.translate('Channel'), icon: DrawToolsChannelIcon },
+    segment:    { id: 'continuous',  text: t.translate('Continuous'), icon: DrawToolsContinuousIcon },
+    fibfan:     { id: 'fibfan',      text: t.translate('Fib Fan'), icon: DrawToolsFibonaccifanIcon },
+    horizontal: { id: 'horizontal',  text: t.translate('Horizontal'), icon: DrawToolsHorizontalIcon },
+    line:       { id: 'line',        text: t.translate('Line'), icon: DrawToolsLineIcon },
+    ray:        { id: 'ray',         text: t.translate('Ray'), icon: DrawToolsRayIcon },
+    rectangle:  { id: 'rectangle',   text: t.translate('Rectangle'), icon: DrawToolsRectangleIcon },
+    tirone:     { id: 'tirone',      text: t.translate('Trend'), icon: DrawToolsTrendIcon },
+    vertical:   { id: 'vertical',    text: t.translate('Vertical'), icon: DrawToolsVerticalIcon },
+};
```
Comment (mahdiaryayi-fs) [inline src/Constant.js:55]: Can we keep vertical alignment for icon field too?

```
export const drawTools = {
    channel:    { id: 'channel',     text: t.translate('Channel'),    icon: DrawToolsChannelIcon },
    segment:    { id: 'continuous',  text: t.translate('Continuous'), icon: DrawToolsContinuousIcon },
    fibfan:     { id: 'fibfan',      text: t.translate('Fib Fan'),    icon: DrawToolsFibonaccifanIcon },
    horizontal: { id: 'horizontal',  text: t.translate('Horizontal'), icon: DrawToolsHorizontalIcon },
    line:       { id: 'line',        text: t.translate('Line'),       icon: DrawToolsLineIcon },
    ray:        { id: 'ray',         text: t.translate('Ray'),        icon: DrawToolsRayIcon },
    rectangle:  { id: 'rectangle',   text: t.translate('Rectangle'),  icon: DrawToolsRectangleIcon },
    tirone:     { id: 'tirone',      text: t.translate('Trend'),      icon: DrawToolsTrendIcon },
    vertical:   { id: 'vertical',    text: t.translate('Vertical'),   icon: DrawToolsVerticalIcon },
};
```

File: src/Constant.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+            { id: 'macd', name: t.translate('MACD'), description: t.translate('There isn\'t any description here.'), icon: IndicatorMacdIcon },
+            { id: 'Price ROC', name: t.translate('Rate of Change'), description: t.translate('There isn\'t any description here.'), icon: IndicatorRateChangeIcon },
+            { id: 'rsi', name: t.translate('Relative Strength Index (RSI)'), description: t.translate('There isn\'t any description here.'), icon: IndicatorRSIIcon },
+            { id: 'stochastics', name: t.translate('Stochastic Oscillator'), description: t.translate('There isn\'t any description here.'), icon: IndicatorStochasticOscillatorIcon },
+            { id: 'Stch Mtm', name: t.translate('Stochastic Momentum Index'), description: t.translate('There isn\'t any description here.'), icon: IndicatorStochasticMomentumIcon },
+            { id: 'Williams %R', name: t.translate('William\'s Percent Range'), description: t.translate('There isn\'t any description here.'), icon: IndicatorWilliamPercentIcon },
+        ],
```
Comment (mahdiaryayi-fs) [inline src/Constant.js:72]: vertical alignment for theses too

File: src/Constant.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    {
+        id: 'others',
+        name: t.translate('Others'),
+        icon: IndicatorCatOtherIcon,
+        items: [
+            { id: 'Alligator', name: t.translate('Alligator'), description: t.translate('There isn\'t any description here.'), icon: IndicatorAlligatorIcon },
+            { id: 'Fractal Chaos Bands', name: t.translate('Fractal Chaos Band'), description: t.translate('There isn\'t any description here.'), icon: IndicatorFractalChaosIcon },
+        ],
+    },
+];
```
Comment (mahdiaryayi-fs) [inline src/Constant.js:114]: and for all of theses

File: src/Constant.js (hunk 1 above)
Comment (shayan-binary-2) [reply src/Constant.js:55]: Sure @mahdipourziaei-binary I'll handle these in another card later as these all, are an improvement.

File: src/Constant.js (hunk 2 above)
Comment (shayan-binary-2) [reply src/Constant.js:72]: Sure @mahdipourziaei-binary I'll handle these in another card later as these all, are an improvement.

File: src/Constant.js (hunk 3 above)
Comment (shayan-binary-2) [reply src/Constant.js:114]: Sure @mahdipourziaei-binary I'll handle these in another card later as these all, are an improvement.

**Pull request merged**